docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,604 @@
+* *Reactive database access with non-reactive drivers*:
+Most JDBC drivers are blocking. Using them in a reactive application is problematic. Helidon DB Client allows the use
+of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service.
+* *Observability*:
+Support for health checks, metrics and tracing was added to all Helidon SE APIs and the Helidon DB Client
+is no exception.
+* *Backpressure*:
+Helidon DB Client performs database operations only when it is requested by the consumer.
+This is propagated all the way to the TCP layer.
+* *Portability between relational database drivers*:
+The DB Client works with native database statements that can be used inline in the code or defined as named statements
+in database configuration. By moving the native query code to configuration files, the Helidon DB Client allows you to
+switch to another database by changing the configuration files, not the code.
+
+== Get started with Helidon DB Client
+
+Nothing better than practice to get in touch with all the features described earlier in this guide.
+This section helps you through your first application using Helidon DB Client. The objective is to
+create a library where customer can bring, get or remove a book.
+
+=== Set up H2 database
+
+A database stores the books from the library. H2 is a java SQL database easy to use and light.
+If H2 is not installed on your machine, here are few steps to quickly download and set it up:
+
+    * Download the latest H2 version from the official website: https://www.h2database.com/html/main.html
+    ** On windows, let guides yourself by the H2 installer.
+    ** On another platform, unzip the downloaded file into your wished directory.
+    ** Only the h2-{latest-version}.jar, located into h2/bin folder, will be needed.
+    * Run the following command to start H2, do not forget to replace `{latest-version}` by your current version.
+
+[source,bash]
+.Open a terminal and enter the following:
+----
+java -jar h2-{latest-version}.jar -webAllowOthers -tcpAllowOthers
+----
+
+This output should be printed:
+
+[source,bash]
+.Terminal output
+----
+Web Console server running at http://127.0.1.1:8082 (others can connect)
+Opening in existing browser session.
+TCP server running at tcp://127.0.1.1:9092 (others can connect)
+PG server running at pg://127.0.1.1:5435 (only local connections)
+----
+
+Open the console at http://127.0.1.1:8082 in your favorite browser. It displays a login window.
+Select `Generic H2 (Server)` from `Saved Settings`. Here is the list of setting, all of them should be set by default.
+
+* Driver Class: org.h2.Driver
+* JDBC URL: jdbc:h2:tcp://localhost/~/test
+* User Name: sa
+* Password:
+
+Password must stay empty. Press connect, the browser displays a web page. The database is correctly set and running.
+
+=== Create a sample SE project
+
+Generate the project sources using the Helidon SE Maven archetype.
```

Comment:
Add a link to `CLI` as a preferred way to create new Helidon apps.

Reply:
This would need to be done for every guide, let's make this change across all of them at once rather than in just one guide.

Reply:
In the last commit I added a link to CLI as Tomas requested. So is it correct or should I remove it and follow Romain approach ? @tomas-langer @romain-grecourt

Reply:
Please follow what @romain-grecourt suggested. Would be great to add an issue for that (if it does not exist already) and mention it on issue of this PR.